Dockerfile won't build gopass starting v1.8.0-alpha.0 up to v1.8.1 & a master as of today.

Dockerfile won't build gopass starting v1.8.0-alpha.0 up to v1.8.1 & a master as of today.

## v1.7.2

```
$ docker build -t gopass github.com/gopasspw/gopass#v1.7.2
Sending build context to Docker daemon  32.94MB
Step 1/9 : FROM golang:1.10-alpine AS builder
 ---> c4b5d89b27f4
Step 2/9 : RUN apk add -U make git gnupg
 ---> Using cache
 ---> 2687e9e57b5b
Step 3/9 : ADD .   /go/src/github.com/justwatchcom/gopass
 ---> d9b9c182b97a
Step 4/9 : WORKDIR /go/src/github.com/justwatchcom/gopass
Removing intermediate container ec66dce0e600
 ---> caee6b08dc91
Step 5/9 : RUN make install
 ---> Running in e67ce5320ff9
/bin/sh: tput: not found
/bin/sh: tput: not found
 [OK]
>> BUILD, version = 1.7.2/39fb91ab, output = gopass
>> bash completion, output = bash.completion
>> fish completion, output = fish.completion
>> zsh completion, output = zsh.completion

>> INSTALL, version = 1.7.2
Removing intermediate container e67ce5320ff9
 ---> c92bd22f34e6
Step 6/9 : RUN chown -R 1000:1000 /go/src/github.com/justwatchcom/gopass
 ---> Running in 55a062b90470
Removing intermediate container 55a062b90470
 ---> f2ad0bfd5bbf
Step 7/9 : ENV HOME /go/src/github.com/justwatchcom/gopass
 ---> Running in b4134b86d6dc
Removing intermediate container b4134b86d6dc
 ---> b998954246de
Step 8/9 : USER 1000:1000
 ---> Running in e6927007fd78
Removing intermediate container e6927007fd78
 ---> 19c9c5c1b015
Step 9/9 : CMD [ "/go/src/github.com/justwatchcom/gopass/gopass" ]
 ---> Running in e8224dcfe2e4
Removing intermediate container e8224dcfe2e4
 ---> 75863692e777
Successfully built 75863692e777
Successfully tagged gopass:latest
```

## v1.8.0-alpha.0

```
$ docker build -t gopass github.com/gopasspw/gopass#v1.8.0-alpha.0
Sending build context to Docker daemon  33.19MB
Step 1/9 : FROM golang:1.10-alpine AS builder
 ---> c4b5d89b27f4
Step 2/9 : RUN apk add -U make git gnupg
 ---> Using cache
 ---> 2687e9e57b5b
Step 3/9 : ADD .   /go/src/github.com/gopasspw/gopass
 ---> 4e2400ff9d5c
Step 4/9 : WORKDIR /go/src/github.com/gopasspw/gopass
Removing intermediate container e42f92d758d4
 ---> 1680263fef82
Step 5/9 : RUN make install
 ---> Running in d95bb5a702b3
/bin/sh: tput: not found
/bin/sh: tput: not found
 [OK]
>> BUILD, version = 1.7.2/9b7563d4, output = gopass# runtime/cgo
exec: "gcc": executable file not found in $PATH
make: *** [Makefile:67: gopass] Error 2
The command '/bin/sh -c make install' returned a non-zero code: 2
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,14 +1,17 @@
 FROM golang:1.10-alpine AS builder
 
-RUN apk add -U make git gnupg
+RUN apk add -U make gcc musl-dev ncurses git
 
 ADD .   /go/src/github.com/gopasspw/gopass
 WORKDIR /go/src/github.com/gopasspw/gopass
 
-RUN make install
+RUN TERM=vt100 make install
 
-RUN chown -R 1000:1000 /go/src/github.com/gopasspw/gopass
-ENV HOME /go/src/github.com/gopasspw/gopass
+FROM alpine:3.7
+RUN apk add -U git gnupg
+COPY --from=0 /go/src/github.com/gopasspw/gopass /usr/bin/
+
+RUN chown -Rh 1000:1000 -- /root
+ENV HOME /root
 USER 1000:1000
-
-CMD [ "/go/src/github.com/gopasspw/gopass/gopass" ]
+ENTRYPOINT [ "/usr/bin/gopass" ]

--- a/docs/setup.md
+++ b/docs/setup.md
@@ -58,6 +58,20 @@ For OpenBSD 6.2 and earlier, install via `go get`.
 Please note that the OpenBSD builds uses `pledge(2)` to disable some syscalls,
 so some features (e.g. version checks, auto-update) are unavailable.
 
+#### Docker
+
+Build it
+
+```
+docker build -t gopass github.com/gopasspw/gopass#master
+```
+
+Use it
+
+```
+alias gopass="docker run --rm -ti -v $HOME:/root gopass"
+```
+
 ### Set up a GPG key pair
 
 gopass depends on the `gpg` program for encryption and decryption. You **must** have a

